classes/class-plugin.php

```diff
@@ -243,7 +242,7 @@ public function frontend_indicator() {
 		$comment = apply_filters( 'wp_stream_frontend_indicator', $comment );
 
 		if ( ! empty( $comment ) ) {
-			echo sprintf( "<!-- %s -->\n", esc_html( $comment ) ); // xss ok.
+			printf( "<!-- %s -->\n", esc_html( $comment ) ); // xss ok.
```

Comment:
Yes, this makes sense. I went ahead and searched for all remaining `xss ok` comments and removed them as well since they seemed to be not needed anymore (`phpcs` does not complain about those lines). Here's the commit: 550377cd

Reply:
Perfect! Thank you! They were so disconcerting.